examples/alanine_dipeptide_tps/AD_tps_1b_trajectory.ipynb

```diff
@@ -204,9 +199,7 @@
   {
    "cell_type": "code",
    "execution_count": 10,
-   "metadata": {
-    "collapsed": false
-   },
+   "metadata": {},
```

Comment:
Before line `219` a template `trajectory`/`snapshot` has to be saved (otherwise it errors out)